docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -75,16 +78,42 @@ autonomy mint skill [OPTIONS] PACKAGE_PATH
 
 ### Examples
 
-Mint the `abstract_abci` skill with dependency IDs 35 and 42 in a local chain:
+Mint the `abstract_abci` skill on a local chain:
 
 ```bash
-autonomy mint --use-local skill --key my_key.txt --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> -d 35 -d 42 ./packages/valory/skills/abstract_abci
+autonomy mint skill --key my_key.txt --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> ./packages/valory/skills/abstract_abci
 ```
 
 Same as above, but using a hardware wallet:
 
 ```bash
-autonomy mint --use-local skill --hwi --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> -d 35 -d 42 ./packages/valory/skills/abstract_abci
+autonomy mint skill --hwi --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> ./packages/valory/skills/abstract_abci
+```
+
+When minting the components the open-autonomy framework uses a `subgraph` for resolving dependencies to their respective on-chain token IDs. If a dependency is not minted you'll get the following error
+
+```
+Error: No on chain registration found for following dependencies
+        - (PACKAGE_TYPE, AUTHOR/NAME:VERSION:PACKAGE_HASH)
+```
+
+The dependency checker uses hashes for resolving token IDs but if you want to use the public IDs to resolve you can use `--skip-hash-check` or `-s` flag on the `autonomy mint` command
+
+```bash
+autonomy mint --skip-hash-check skill --hwi --nft <nft_ipfs_hash_or_image_path> --owner <owner_address> ./packages/valory/skills/abstract_abci
+```
+
+When using public ID for resolving token IDs you might get prompted to select between token IDS since a package can be minted multiple times with different versions. The prompt will look like this
+
+```
+Multiple dependencies found for <public_id> of type <package_type>
+Please choose which dependency to use (token_id_0, token_id_1, ...): 
+```
+
+You can avoid the prompts and just use the latest dependencies by using `-l, --latest-dependencies` flag
```

Comment:
This information should be placed below the main description of the command, and the example briefly mention that we are using this flag.